From c03003a561f9c9fefb741d017778749aa6c7deeb Mon Sep 17 00:00:00 2001
From: Ilya Shlyakhter <ilya_shl@alum.mit.edu>
Date: Tue, 3 Mar 2020 19:59:18 -0500
Subject: [PATCH] fix long paths in locales

---
 runshell | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/runshell b/runshell
index 7ef1601..dd45749 100755
--- a/runshell
+++ b/runshell
@@ -128,17 +128,10 @@ unset LD_PRELOAD
 ORIG_LOCPATH="$LOCPATH"
 export ORIG_LOCPATH
 if [ -z "${LOCPATH+set}" ] && [ -z "$GIT_ANNEX_PACKAGE_INSTALL" ]; then
-	LOCPATH="$HOME/.cache/git-annex/locales/$(echo "$base" | tr / _ )"
+
+	LOCPATH="$HOME/.cache/git-annex/locales/$(echo "$base" | cksum | tr ' ' _)_$(stat -L --printf='%D_%i_%Z' $0)"
 	export LOCPATH
 	
-	# Clean up locale caches when their standalone bundle no longer exists.
-	for localecache in $HOME/.cache/git-annex/locales/*; do
-		cachebase=$(cat "$localecache/base" 2>/dev/null || true)
-		if [ ! -d "$cachebase" ] || ! cmp "$localecache/buildid" "$cachebase/buildid" >/dev/null 2>&1 ; then
-			rm -rf "$localecache" 2>&1 || true
-		fi
-	done
-	
 	# If the locale cache for this bundle is out of date, refresh it.
 	if [ -e "$LOCPATH/buildid" ] && ! cmp "$LOCPATH/buildid" "$base/buildid" >/dev/null 2>&1 ; then
 		rm -rf "$LOCPATH"
-- 
2.25.0

