From 7020dc022d84e0b6093ff92451c16812c961d156 Mon Sep 17 00:00:00 2001
From: Ilya Shlyakhter <ilya_shl@alum.mit.edu>
Date: Tue, 10 Mar 2020 11:29:28 -0400
Subject: [PATCH] new patch to add fdo-notify and magic

---
 stack.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/stack.yaml b/stack.yaml
index 487ccfa..f718cdc 100644
--- a/stack.yaml
+++ b/stack.yaml
@@ -15,6 +15,8 @@ flags:
 packages:
 - '.'
 extra-deps:
+ - fdo-notify-0.3.1
+ - magic-1.1
  - IfElse-0.85
  - aws-0.21.1
  - bloomfilter-2.0.1.0
-- 
2.23.1

