From 06f9f029937673d29ef850b69913f45849178cb5 Mon Sep 17 00:00:00 2001
From: Ilya Shlyakhter <ilya_shl@alum.mit.edu>
Date: Mon, 19 Aug 2019 15:20:34 -0400
Subject: [PATCH] added extra deps for fdo notify and magic

---
 stack.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/stack.yaml b/stack.yaml
index d97bf2f..17d4c03 100644
--- a/stack.yaml
+++ b/stack.yaml
@@ -15,6 +15,8 @@ flags:
 packages:
 - '.'
 extra-deps:
+- fdo-notify-0.3.1
+- magic-1.1
 - IfElse-0.85
 - aws-0.21.1
 - bloomfilter-2.0.1.0
-- 
2.20.1

