From 1c88baa9da2863c1866ce311f9b67474dab7f669 Mon Sep 17 00:00:00 2001
From: Ilya Shlyakhter <ilya_shl@alum.mit.edu>
Date: Thu, 4 Jun 2020 18:39:03 -0400
Subject: [PATCH] added extra-deps for magic

---
 stack.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/stack.yaml b/stack.yaml
index 487ccfad2..e1c362373 100644
--- a/stack.yaml
+++ b/stack.yaml
@@ -15,6 +15,7 @@ flags:
 packages:
 - '.'
 extra-deps:
+ - magic-1.1 
  - IfElse-0.85
  - aws-0.21.1
  - bloomfilter-2.0.1.0
-- 
2.23.3

