From 4df0d08572859f6e58da70f76aecffa0aeab8f43 Mon Sep 17 00:00:00 2001
From: Ilya Shlyakhter <ilya_shl@alum.mit.edu>
Date: Thu, 14 Mar 2019 13:13:45 +0000
Subject: [PATCH 1/2] add wcwidth to extra-deps

---
 stack.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/stack.yaml b/stack.yaml
index 5653705..1e137e7 100644
--- a/stack.yaml
+++ b/stack.yaml
@@ -15,6 +15,7 @@ flags:
 packages:
 - '.'
 extra-deps:
+- wcwidth-0.0.2
 - IfElse-0.85
 - aws-0.20
 - bloomfilter-2.0.1.0
-- 
2.20.1

