From d70554c563ab09b2c52324a78da693a987dd201f Mon Sep 17 00:00:00 2001
From: Ilya Shlyakhter <ilya_shl@alum.mit.edu>
Date: Thu, 14 Mar 2019 13:14:17 +0000
Subject: [PATCH 2/2] change resolver to lts-11.22

---
 stack.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stack.yaml b/stack.yaml
index 1e137e7..1cb5b50 100644
--- a/stack.yaml
+++ b/stack.yaml
@@ -25,4 +25,4 @@ extra-deps:
 - sandi-0.5
 explicit-setup-deps:
   git-annex: true
-resolver: lts-13.6
+resolver: lts-11.22
-- 
2.20.1

